src/scripts/components/story/story.tsx

```diff
@@ -40,6 +42,16 @@ const Story: FunctionComponent = () => {
     return null;
   }
 
+  const getRightSideComponent = (slide: Slide, story: StoryType) => {
+    if (slide.images) {
+      return <StoryMedia images={slide.images} storyId={story.id} />;
+    } else if (slide.videoId) {
+      return <StoryVideo videoId={slide.videoId} />;
+    }
+
+    return <Globes />;
```

Comment:
hmm I don't think the globe transitions will work this way, could be that they jump to the next position instead of a flyto but let's see...